pom.xml

```
@@ -12,12 +12,14 @@
 
   <groupId>io.jenkins.plugins</groupId>
   <artifactId>github-checks</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
   <description>Allows users to publish GitHub checks</description>
 
   <properties>
     <java.level>8</java.level>
+    <revision>0.1.0</revision>
```

I'm not sure if it makes sense to publish a beta? Then someone can install it from the experimental update center.
```suggestion
    <revision>0.1.0-beta-1</revision>
```

---

can always add the beta in at release time?